synapse/models/inet.py

```diff
@@ -332,13 +331,16 @@ def repr(self, valu):
         return valu
 
 class InetMod(CoreModule):
+
     def initCoreModule(self):
         # add an inet:defang cast to swap [.] to .
         self.core.addTypeCast('inet:defang', castInetDeFang)
         self.core.addTypeCast('inet:ipv4:cidr', ipv4cidr)
         self.onFormNode('inet:fqdn', self.onTufoFormFqdn)
         self.onFormNode('inet:passwd', self.onTufoFormPasswd)
 
+        self.core.on('node:prop:set', self.onSetFqdnSfx, prop='inet:fqdn:sfx')
```

Comment:
core events are already distributed to coremodules.

Reply:
does this still need to be addressed?